frontend/src/pages/Search.vue

```diff
@@ -93,25 +90,22 @@ export default {
           total: data.total,
           duration: data.duration,
         }
-        for (let doctype in data.results) {
-          let group = null
-          if (doctype === 'GP Discussion') {
-            group = 'Discussions'
-          } else if (doctype === 'GP Task') {
-            group = 'Tasks'
-          } else if (doctype === 'GP Page') {
-            group = 'Pages'
-          }
-          if (!group) {
-            continue
+        for (let group in data.results) {
```

Comment:
probably results should be a list and not be grouped by doctype. any data transformation should be done on the client.

Reply:
Yeah, groupby should default to `False`. Changed it back now.